build/common.sh

```diff
@@ -198,10 +143,96 @@ function kube::build::verify_prereqs() {
 # ---------------------------------------------------------------------------
 # Utility functions
 
+function kube::build::docker_available_on_osx() {
+  if [[ -z "${DOCKER_HOST}" ]]; then
+    kube::log::status "No docker host is set. Checking options for setting one..."
+
+    if [[ -z "$(which docker-machine)" && -z "$(which boot2docker)" ]]; then
+      kube::log::status "It looks like you're running Mac OS X, and docker-machine is nowhere to be found."
```

Comment:
You check both docker-machine and boot2docker here, so maybe the log message should include both.